src/components/BenefitPlanBeneficiariesSearcher.js

```diff
@@ -1,24 +1,33 @@
-import React from "react";
+import React, { useState, useCallback, useEffect, useRef } from "react";
 import {injectIntl} from "react-intl";
-import {formatMessageWithValues, Searcher} from "@openimis/fe-core";
+import {formatMessage, formatMessageWithValues, Searcher, downloadExport} from "@openimis/fe-core";
 import {bindActionCreators} from "redux";
 import {connect} from "react-redux";
-import {fetchBeneficiaries} from "../actions";
-import {DEFAULT_PAGE_SIZE, ROWS_PER_PAGE_OPTIONS} from "../constants";
+import {fetchBeneficiaries, downloadBeneficiaries} from "../actions";
+import {DEFAULT_PAGE_SIZE, ROWS_PER_PAGE_OPTIONS, RIGHT_BENEFICIARY_SEARCH} from "../constants";
 import BenefitPlanBeneficiariesFilter from "./BenefitPlanBeneficiariesFilter";
+import {Button,
```

Comment:
```
import {
  Button,
  Dialog,
  DialogActions,
  DialogTitle,
} from "@material-ui/core";
```
You can fix the way of declaring imports to satisfy some standards defined by eslint rules regarding imports.